Added support of CephFS volumes

loicalbertin opened the pull request:
Closes #11

loicalbertin committed:
Added support of CephFS volumes

laurentganne commented on src/main/resources/csar/tosca.yml:
```diff
@@ -1135,6 +1135,43 @@ data_types:
+        entry_schema:
+          type: string
+        description: |
+          Required: Monitors is a collection of Ceph monitors. More info: https://github.com/kubernetes/examples/blob/master/volumes/cephfs/README.md
```
May not be needed to have the description start with "Required: " or "Optional:" as it is seen in the UI which is taking into account the value of the property "required" (and other data types descriptions don't use this convention)

loicalbertin replied:
You're right! Updated.

laurentganne reviewed (approved):
Looks good !

loicalbertin committed:
Update tosca.yml

Take Laurent's comment into account, removed optional/required in parameters description
as this is already specified in TOSCA
